src/GUI/ErikEJ.EntityFrameworkCore.SqlServer.Dacpac/DacPacEngine/DacpacTableListBuilder.cs

```diff
@@ -25,7 +25,7 @@ public DacpacTableListBuilder(string dacpacPath)
             _dacpacPath = dacpacPath;
         }
 
-        public List<Tuple<string, bool, List<string>>> GetTableDefinitions()
+        public List<Tuple<string, bool, List<string>, List<string>>> GetTableDefinitions()
```

Comment:
While I was modeling this feature, I modified the result as List<Tuple<string, bool, List<Tuple<string, bool>>>>. Just to let you know. I like your solution too. Mine was just closer to the TableModel/ColumnModel shape.